flocx_market/api/bid.py

```diff
@@ -2,36 +2,47 @@
 from flask import request
 from flocx_market.objects import bid
 
+import flocx_market.api.app as application
```

Comment:
Minor issue thing that doesn't need to be fixed, but I'd personally just use 'from flocx_market.api import app` and then call `app.create_context()`.